content/docs/tasks/traffic-management/egress/egress-sample.txt

```diff
@@ -0,0 +1,23 @@
+# $snippet SNIP1
+$ kubectl exec -it $SOURCE_POD -c sleep -- curl -I https://www.google.com | grep  "HTTP/"; kubectl exec -it $SOURCE_POD -c sleep -- curl -I https://edition.cnn.com | grep "HTTP/"
+command terminated with exit code 35
+command terminated with exit code 35
+# $endsnippet
+
+# $snippet SNIP2
```

Comment:
This snippet is a proper command, but the first line has an extra `$` character at the beginning, so this one also can't just be run as a test command.

Reply:
Oh, I hadn't thought about the $ stuff. Not sure how to handle that :-(